app/forms/news_article_form.rb

```diff
@@ -1,5 +1,14 @@
 class NewsArticleForm < NodeForm
   feature Reform::Form::MultiParameterAttributes
+  property :name, presence: true
+  property :short_summary, presence: true
+  property :summary, presence: true
+  property :section_id
+  property :section_ids
   property :release_date, multi_params: true
   validates :release_date, presence: true
+  validates :name, presence: true
+  validates :section_id, presence: true
+  validates :short_summary, presence: true
+  validates :summary, presence: true
```

Comment:
I'll remove the dupes, but I think presence should be validated. I will put this on the base NodeForm class, particularly for things like `:name` which _should_ have a value provided.